gulp.md

```diff
@@ -0,0 +1,16 @@
+# What's it do?
+`gulpfile.js` looks through `ACM_General/app/static/app/scss/` for sass
+files to compile then drops them into `ACM_General/app/static/app/css/`
```

Comment:
compiles the sass files, and then drops the results into `ACM_Genera/*/static/*/css/` with the same full path as the `scss` folder.